plugins/modules/omevv_firmware_repository_profile.py

```diff
@@ -396,6 +425,98 @@ def execute(self):
             self.module.exit_json(msg=CHANGES_NOT_FOUND_MSG, diff={"before": {}, "after": {}}, changed=False)
 
 
+class ResyncFirmwareRepositoryProfile(FirmwareRepositoryProfile):
+
+    def __init__(self, module, rest_obj, rest_ome_obj):
+        self.module = module
+        super().__init__(module, rest_obj)
+        self.rest_obj = rest_obj
+        self.ome_obj = rest_ome_obj
+
+    def check_plugin_availability(self):
+        ump_plugin = 0
+        details = self.ome_obj.invoke_request("GET", "PluginService/Plugins")
+        resp = details.json_data["value"]
+        for item in resp:
+            if item.get("Name") == "Update Manager":
+                ump_plugin = 1
+                break
+        if not ump_plugin:
+            self.module.fail_json(msg=UMP_PLUGIN_NOT_FOUND_MSG, failed=True)
+
+    def check_mode_support(self):
+        omevv_profiles = self.omevv_profile_obj.get_firmware_repository_profile()
+        res = self.ome_obj.invoke_request("GET", "UpdateManagementService/Repositories")
```

Comment:
Let's put  "UpdateManagementService/Repositories" into a variable as it is the API endpoint.

Reply:
done